tests/integration/activity_test.go

```diff
@@ -29,7 +34,7 @@ func TestActivity_Starring(t *testing.T) {
 	}
 
 	// first, check if already starred google/go-github
```

Comment:
Now that owner and repo are factored out into constants outside, it makes sense to update the comment to not mention google/go-github repo specifically, but rather use more broad terms. For example:

```
// first, check if already starred the target repository
```